xapian-core/tests/api_queryparser.cc

```diff
@@ -1743,19 +1743,19 @@ DEFINE_TESTCASE(qp_filesizerange1, !backend) {
     for (const test *p = test_value_filesizerange1_queries; p->query; ++p) {
     string expect, parsed;
     if (p->expect)
-	expect = p->expect;
+	   expect = p->expect;
     else
-	expect = "parse error";
+	   expect = "parse error";
     try {
-	Xapian::Query qobj = qp.parse_query(p->query);
-	parsed = qobj.get_description();
-	expect = string("Query(") + expect + ')';
+	   Xapian::Query qobj = qp.parse_query(p->query);
+	   parsed = qobj.get_description();
+	   expect = string("Query(") + expect + ')';
     } catch (const Xapian::QueryParserError &e) {
-	parsed = e.get_msg();
+	   parsed = e.get_msg();
     } catch (const Xapian::Error &e) {
-	parsed = e.get_description();
+	   parsed = e.get_description();
     } catch (...) {
-	parsed = "Unknown exception!";
+	   parsed = "Unknown exception!";
     }
```

Comment:
@shivanshuiitg I think you need to spend some time figuring out how to configure your editor for Xapian's indentation and alignment approach. If you get the config right, you shouldn't have to worry about indentation in your code very much, as it'll do most of the work for you. The amount of code you'd need to write over the summer is enough that this will save you a lot of time.

The main rules are: each level of indentation is by four positions, and indentation whitespace should be tabs (of eight positions) followed by spaces.